development/development.env

```diff
@@ -21,11 +19,24 @@ NAUTOBOT_REDIS_PORT=6379
 # NAUTOBOT_REDIS_SSL=True
 
 # Nautobot DB Connection Environment Variables
-NAUTOBOT_DB_HOST=db
 NAUTOBOT_DB_NAME=nautobot
 NAUTOBOT_DB_USER=nautobot
+NAUTOBOT_DB_HOST=db
 NAUTOBOT_DB_TIMEOUT=300
 
+# Use them to overwrite the defaults in nautobot_config.py
+# NAUTOBOT_DB_ENGINE=django.db.backends.postgresql
+# NAUTOBOT_DB_PORT=5432
+
+# Needed for Postgres should match the values for Nautobot above
+POSTGRES_USER=${NAUTOBOT_DB_USER}
+POSTGRES_DB=${NAUTOBOT_DB_NAME}
+
+# Needed for MYSQL should match the values for Nautobot above
+MYSQL_USER=${NAUTOBOT_DB_USER}
+MYSQL_DATABASE=${NAUTOBOT_DB_NAME}
+MYSQL_ROOT_HOST=%
+
```

Comment:
Remove as irrelevant to this App

Reply:
Would rather keep it here, and resolve this by customizing the template.